packages/manager/modules/navbar/src/notifications-menu/constants.js

```diff
@@ -6,9 +6,12 @@ export const ANIMATED_STATUS = ['incident', 'error', 'warning'];
 
 export const REFRESH_TIME = 60000;
 
+export const MAX_NOTIFICATIONS_TO_SOW = 10;
```

Comment:
`SOW` ?

I feel like `MAX_NOTIFICATIONS`is enough, don't you ?

Reply:
yes agree